pkg/apis/extensions/validation/validation.go

```diff
@@ -768,6 +769,26 @@ func validatePSPRunAsUser(fldPath *field.Path, runAsUser *extensions.RunAsUserSt
 	return allErrs
 }
 
+// validatePSPRunAsGroup validates the RunAsGroup fields of PodSecurityPolicy.
+func validatePSPRunAsGroup(fldPath *field.Path, runAsGroup *extensions.RunAsGroupStrategyOptions) field.ErrorList {
```

Comment:
Could you add a unit tests for this function? (Yes, I know that `validatePSPRunAsUser` doesn't have the tests, but I'd like to not copy a bad practice in a new code.)

Reply:
done